doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -0,0 +1,512 @@
+:orphan:
+
+.. _ug_nrf54h20_ironside_se:
+
+IronSide Secure Element
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The IronSide Secure Element (IronSide SE) is a firmware for the Secure Domain of the nRF54H20 SoC that provides security features.
+It is provided as a precompiled binary, part of the nRF54H20 SoC bundle.
+
+IronSide SE provides the following features:
+
+* Global memory configuration (via UICR.MPCCONF)
```

Comment:
```suggestion
* Global memory configuration (through UICR.MPCCONF)
```

Reply:
IronSide doesn't currently apply UICR.MPCCONF. We have only defined the format/interface.

Reply:
We have to clarify how to configure UICRs then @jonathannilsen